pyproject.toml

```diff
@@ -197,7 +198,7 @@ max-line-length = 99
 [tool.tox]
 legacy_tox_ini = """
 [tox]
-envlist = py310, py311, py312, lint, type-check, docs, build
+envlist = py310, py312, py313, lint, type-check, docs, build
```

Comment:
there is no need to remove the tox environment for Python 3.11, even if we wont test against it on CI